### DISCUSSION TITLE
NVR Hikvision

### DISCUSSION THREAD

**Discussion by vipersinu2** [gtxaspec/wz_mini_hacks#479] [Q&A]
I was able to setup the RTSP feed on camera, I configured the Hikvision NVR for the RTSP feed, The NVR says camera is connected and online, but the screen is black when viewing, The VLC player works well for testing, 
My question is, if I setup the POE USB on the camera, and connect to the NVR directly do you think I can get video on the Hikvision? Or do you think Wyze will never work with a Hikvision NVR?

**Comment by Bob-AOL** [comment]
It will work. I have several v3s working with an Amcrest NVR, both wired and wireless and both Wyze Beta RTSP and wz_mini RTSP versions. Their resoltution isn't detected correctly, but they still work and look fine. Still a work in progress to identify why they are detected as QQVGA but still display at what appears to be at least 720p if not 1080p which VLC player says they are. Regardless, they work with the Amcrest NVR -- and Blue Iris simultaneously.

The NVR will be picky about the parameters it wants to see from the source regardless of networked or wireless -- beginning with the correct operating parameters of the camera itself. Address, resolution, encoding, frame and bit rate...and username/password.

It is also important to set up the cameras from a console connection to the Hik; not from an app or browser. User credentials can get mixed results when too many authentications are involved.
